src/compiler/checker.ts

```diff
@@ -15247,6 +15248,7 @@ namespace ts {
          *
          * @param openingLikeElement a non-intrinsic JSXOPeningLikeElement
          * @param shouldIncludeAllStatelessAttributesType a boolean indicating whether to include all attributes types from all stateless function signature
+         * @param sourceAttributesType Is the attributes type the user has actually passed, and is used to create interfences in the target type if present

```

Comment:
delete "has actually"
typo "inferences"

Reply:
As usual, I'm not convinced these `@param` tags are useful. The `@return` is, though.

Well, the second sentence for `elementType` is useful, I guess.